spec/site_prism/element_spec.rb

```diff
@@ -33,6 +33,28 @@
         expect(subject).not_to have_element_two
       end
 
+      context 'when other classes coincidentally have the overlapping methods defined' do
+        subject { anonymous_test_class.new }
+
+        let!(:anonymous_site_prism_class) do # rubocop:disable RSpec/LetSetup
```

Comment:
this all is provided from the shared context

Reply:
I'm not sure what you mean here? This is a method of defining classes that avoids constant leakage (https://fili.pp.ru/leaky-constants.html); I'm unsure how it relates to the shared context.